### ISSUE TITLE
Validity style and script needs protocols?

### ISSUE THREAD

**Issue by fahmi182** [ampproject/amphtml#1895]
Hi all,

i want to asking, validity in AMP needs protocols? if our site load script or style with:

    src=//cdn.ampproject.org/v0.js

not valid?

    Fix invalid tag attribute values. AMP pages with invalid attribute values may appear in Google Search results without AMP-specific display features.

we must using protocols to get valid?

**Comment by Gregable** [comment]
Yes, for the ampproject.org files, you must specify a full, non-relative URL. `https://cdn.ampproject.org/v0.js` in this case.

**Comment by fahmi182** [comment]
if we used wordpress with cloudflare and ssl content fixer (plugin), all protocols will removed, and i think validity needs to improve to accept non-protocols. :)

**Comment by cramforce** [comment]
@fahmi182 It would be wrong. We do not serve cdn.ampproject.org on http. Please file a bug with cloudflare.

See also why protocol relative URLs should no longer be used http://www.paulirish.com/2010/the-protocol-relative-url/
TL;dr is that they've been used for spying and DOS attacks.

**Comment by fahmi182** [comment]
I think you missunderstand, i mean.. I have a website using wordpress, cloudflare, ssl, and i fixed all source (ssl content fixed) from non-https to "//" and now all link become "//"

Example :

//cdn.ampproject.org

With no http or https
